pkg/prometheus/promcfg.go

```diff
@@ -747,22 +762,6 @@ func (cg *configGenerator) generateProbeConfig(
 			cfg = append(cfg, cg.generateK8SSDConfig(selectedNamespaces, apiserverConfig, basicAuthSecrets, kubernetesSDRoleIngress))
 		}
 
-		// Relabelings for prober.
```

Comment:
why would this not be needed for ingress targets?

Reply:
Thanks for the review, I will add it back.